src/renderer/src/components/TransactionActivityList/TransactionActivityListDateRange.tsx

```diff
@@ -19,10 +21,16 @@ export const TransactionActivityListDateRange = ({
   onSelectDateFrom,
   onSelectDateTo,
 }: TProps) => {
+  const { language } = useLanguageSelector()
   const { t } = useTranslation('components', { keyPrefix: 'transactionActivityList.dateRange' })
 
   return (
-    <div className="flex h-9 w-56 min-w-56 max-w-56 items-center justify-center gap-x-0.5 rounded bg-asphalt px-2">
+    <div
+      className={StyleHelper.mergeStyles('flex h-9 items-center justify-center gap-x-0.5 rounded bg-asphalt px-2', {
```

Comment:
You can simply set "min-w-56 max-w-72" instead of setting a different width for each language